graduation-project-tips.md

```diff
@@ -0,0 +1,93 @@
+# Ten Things they don't tell you about your graduation project
+
+In this document, I try to list the most important tips and tricks (in my opinion) in the hopes of helping future CCEians by sharing my experience with the graduation project and how to get the most out of it (hopefully).
+
+## Tip #1: Team Comes First :two_men_holding_hands:
+
+The team is the **most important factor** of any project, a team determines how the project starts, how the project ends and pretty much everything in between. 
+You must be comfortable to be yourself with your team and feel among friends, if this condition is not achieved, there will be fights and fights is not good for work.
+
+This is based on true events, it happened in my batch, in previous batches and will happen sadly in future batches (unless everyone reads/applies this tip hopefully).
```

Comment:
Also too gloomy :) Let's send a more positive message and focus on action points.